netdata.spec.in

```diff
@@ -47,18 +47,25 @@ make %{?_smp_mflags}
 rm -rf "%{buildroot}"
 make %{?_smp_mflags} install DESTDIR="%{buildroot}"
 find "%{buildroot}" -name .keep -exec rm {} \;
+touch %{buildroot}/%{_sysconfdir}/netdata/netdata.conf
+%if 0%{?fedora} || 0%{?rhel} >= 7 
+install -m 0644 -D system/netdata-systemd ${RPM_BUILD_ROOT}%{_unitdir}/netdata.service
+%endif
 
 %files
 %attr(-, netdata, netdata) %dir %{_localstatedir}/cache/%{name}/
 %attr(-, netdata, netdata) %dir %{_localstatedir}/log/%{name}/
+%attr(-, netdata, netdata) %{_datadir}/%{name}/
 %config(noreplace) %{_sysconfdir}/%{name}/
 %if 0%{?rhel}
 %{_sbindir}/%{name}
 %else
 %{_bindir}/%{name}
 %endif
-%{_datadir}/%{name}/
```

Comment:
I do not follow. the netdata-installer.sh was always align to packaging. in best case it should copy the /usr/share files and create an instance at /var/lib to be used. otherwise, you should use the static data from /usr/share and only store the files that you write into in /var/lib, if you must have one directory for all, create /var/lib and symlink /usr/share for static data.